todo_app/data/trello_items.py

```diff
@@ -9,7 +9,8 @@ def get_api_query_params():
     return {'key': os.getenv("API_KEY"), 'token': os.getenv("API_TOKEN")}
 
 def get_items():
-    params = get_api_query_params() | {'cards': 'open'}
```

Comment:
I had some real struggles trying to install python 3.10 correctly. It appeared to install correctly but when running poetry I would get errors missing the ssl module. May come back to this if I have time later down the line.

Reply:
Sounds like sensible prioritisation to me; if you want to share at some point how you set up Python3.10 so we can take a look and see if we can advise then feel free to!